### DISCUSSION TITLE
[Noob Question] How do I check my changes?

### DISCUSSION THREAD

**Discussion by hasnainzeenwala** [pytest-dev/pytest#10033] [Q&A] [answered]
Hi! This is my first time trying to contribute to open source. I've forked and cloned the repo and I'm trying to figure out how do I check my changes? Like how do I import the pytest code modified by me locally into some python program to check if my changes have had the desired effect.

**Comment by nicoddemus** [comment, marked as answer]
Hi @hasnainzeenwala,

You can install pytest into development mode by calling `pip install -e .` from pytest's directory from within an activate virtual environment.

**Reply by hasnainzeenwala** [reply]
Thanks!!
